### DISCUSSION TITLE
Beutification bug?

### DISCUSSION THREAD

**Discussion by uri613** [phcode-dev/phoenix#1802] [General]
Within a PHP file, I have a line of HTML that contains:
<?= htmlspecialchars($row['itemTitle'] ?? '', ENT_QUOTES) ?>
When I 'beautify' the code in Phoenix, it gets broken up like this:
      <textarea class='fieldInput fullWidth swiper-no-swiping' id='itemDescription' name='itemDescription' cols=50 onfocus='prepChars(this);' onblur='prepChars();' oninput='auto_grow(this);maxChars(this,1000)'><?= htmlspecialchars(
            $d ?? "",
            ENT_QUOTES
        ) ?></textarea>
Which breaks the HTML functionality completely!
I've confirmed that it happens multiple times to similar code, making my 'beautify' deadly. I don't know where else to report this...

**Comment by abose** [comment]
That is valid html, attributes can be split into multiple lines.

**Comment by abose** [comment]
That is valid html, attributes can be split into multiple lines.  @uri613 what problem happens when it gets split?

**Reply by charlypa** [reply]
@uri613 
Try selecting following in menu `View -> Word Wrap`
